internal/beater/otlp/grpc_test.go

```diff
@@ -171,28 +179,21 @@ func TestConsumeLogsGRPC(t *testing.T) {
 	assert.Len(t, batches[0], 1)
 	assert.Len(t, batches[1], 1)
 
-	actual := map[string]interface{}{}
-	monitoring.GetRegistry("apm-server.otlp.grpc.logs").Do(monitoring.Full, func(key string, value interface{}) {
-		actual[key] = value
+	monitoringtest.ExpectContainOtelMetrics(t, reader, map[string]any{
+		"grpc.server.request.count":         2,
+		"grpc.server.response.valid.count":  1,
+		"grpc.server.response.count":        2,
+		"grpc.server.response.errors.count": 1,
 	})
```

Comment:
It appears that here and in other places the metric names have changed, i.e. in this case from `apm-server.otlp.grpc.logs.*` to `apm-server.grpc.server.*`. Is that intentional?